mt32emu/src/mt32emu.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
```

Comment:
I'm not sure if this is correct, as I haven't figured yet how CMAKE_INSTALL_LIBDIR works. The docs say we should include(GNUInstallDirs) module first to have several standard paths defined, but I stumbled with built-in default "lib" set there for all systems I tried regardless.

To configure the target library dir for installation we have a custom variable LIB_INSTALL_DIR defined elsewhere in this CMakeLists.txt, so we probably can use just that without GNUInstallDirs inclusion.

Reply:
Oh, on Fedora 33 CMAKE_INSTALL_LIBDIR is magically set to lib64 :)
Not bad, albeit it's not really what I expected from GUNInstallDirs module :/

Reply:
I've revised the template a bit and used your custom variable LIB_INSTALL_DIR instead. I didn't notice this but @CMAKE_INSTALL_LIBDIR@ did not fill in anything, as, from your answer, that module would need to be included first.

Also good points on the include dir and cflags setting, I changed that, too.

Should I squash this to one commit? Or should we try with the GNUInstallDirs module and use CMAKE_INSTALL_LIBDIR? (and then also fill in the include dir with CMAKE_INSTALL_INCLUDEDIR?

Reply:
Atm, I'm a bit disappointed by GNUInstallDirs. It is supposed to fill in many standard paths but instead quite a bit of variables are merely empty. That's rather worrying. I've just added platform-specific library dirs into my build scripts, this works well :)

Thanks for donation ;)